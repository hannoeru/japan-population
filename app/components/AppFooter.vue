<script setup lang="ts">
const isDark = useDark()
const toggleDark = useToggle(isDark)
</script>

<template>
  <footer class="flex items-center justify-center gap-3 p-6 pb-8 text-lg">
    <button title="Toggle Dark Mode" @click="toggleDark()">
      <span class="i-ph-sun-duotone dark:i-ph-moon-stars-duotone flex" />
    </button>
    <a href="https://github.com/hannoeru/japan-population" target="_blank" title="GitHub Source Code">
      <span class="i-ph-github-logo-duotone flex" />
    </a>
  </footer>
</template>
