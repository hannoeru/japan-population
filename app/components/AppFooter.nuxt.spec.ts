import { describe, expect, it } from 'vitest'

import { mount } from '@vue/test-utils'
import AppFooter from './AppFooter.vue'

describe('appFooter', () => {
  it('matches snapshot', async () => {
    const component = mount(AppFooter)
    expect(component.html()).toMatchSnapshot()
  })
})
