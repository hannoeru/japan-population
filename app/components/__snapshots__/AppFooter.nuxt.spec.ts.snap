// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`appFooter > matches snapshot 1`] = `"<footer class="flex items-center justify-center gap-3 p-6 pb-8 text-lg"><button title="Toggle Dark Mode"><span class="i-ph-sun-duotone dark:i-ph-moon-stars-duotone flex"></span></button><a href="https://github.com/hannoeru/japan-population" target="_blank" title="GitHub Source Code"><span class="i-ph-github-logo-duotone flex"></span></a></footer>"`;
